tools/realsense-viewer/realsense-viewer.cpp

```diff
@@ -295,43 +298,46 @@ int main(int argc, const char** argv) try
     std::vector<device> connected_devs;
     std::mutex m;
 
-#ifdef BUILD_SHARED_LIBS
-    // Configure the logger
-    el::Configurations conf;
-    conf.set( el::Level::Global, el::ConfigurationType::Format, "[%level] %msg" );
-    conf.set( el::Level::Info, el::ConfigurationType::Format, "%msg" );
-    conf.set( el::Level::Debug, el::ConfigurationType::Enabled, "false" );
-    el::Loggers::reconfigureLogger( "default", conf );
-    // Create a dispatch sink which will get any messages logged to EasyLogging, which will then
-    // post the messages on the viewer's notification window.
-    class viewer_model_dispatcher : public el::LogDispatchCallback
-    {
-    public:
-        rs2::viewer_model * vm = nullptr;  // only the default ctor is available to us...!
-    protected:
-        void handle( const el::LogDispatchData* data ) noexcept override
+#ifdef BUILD_EASYLOGGINGPP
+    #ifdef BUILD_SHARED_LIBS
+        // Configure the logger
```

Comment:
The code itself shouldn't be indented with the #ifdef -- it should match the indentation of the surrounding code.